doc/modules/model_evaluation.rst

```diff
@@ -353,6 +353,23 @@ In the multilabel case with binary label indicators: ::
     for an example of accuracy score usage using permutations of
     the dataset.
 
+.. _cohen_kappa:
+
+Cohen's kappa
+-------------
+
+The function :func:`cohen_kappa` computes Cohen's kappa statistic.
+This measure is intended to compare labelings by different human annotators,
+not a classifier versus a ground truth.
```

Comment:
IME, it's simply never done, but I'm not sure why. I thought it might be because the random guessing baseline isn't appropriate, but actually it's a pretty smart correction. In particular, note that a model that always assigns the majority class will always achieve kappa=0 (or kappa=nan if the ground truth has one label):

```py
# Example data
>>> y1 = array([1,1,1,0,0,0,0,0,0,0,1,0,0,2,2,0,0,0,0])
>>> y2 = zeros_like(y1)
>>> confusion = confusion_matrix(y1, y2)
>>> P = confusion / float(confusion.sum())
>>> p_observed = np.trace(P)
>>> p_expected = np.dot(P.sum(axis=0), P.sum(axis=1))
>>> p_observed, p_expected
(0.68421052631578949, 0.68421052631578949)
>>> (p_observed - p_expected) / (1 - p_expected)
0.0
```

This occurs because the trace will always equal `P[0,0]` in this case, and the dot product will give the same result, because `P` will have one non-zero column (or row):

```py
>>> P
array([[ 0.68421053,  0.        ,  0.        ],
       [ 0.21052632,  0.        ,  0.        ],
       [ 0.10526316,  0.        ,  0.        ]])
```

Anyway, something being intended for a purpose doesn't mean it cannot be used for other purposes :)